{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "n/a",
                    "vendor": "n/a",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "n/a"
                        }
                    ]
                }
            ],
            "datePublic": "2008-11-12T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "PythonScripts in Zope 2 2.11.2 and earlier, as used in Conga and other products, allows remote authenticated users to cause a denial of service (resource consumption or application halt) via certain (1) raise or (2) import statements."
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "n/a",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2009-09-01T09:00:00",
                "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
                "shortName": "mitre"
            },
            "references": [
                {
                    "name": "[Zope] 20080812 Script (Python) insecure ?",
                    "tags": [
                        "mailing-list",
                        "x_refsource_MLIST"
                    ],
                    "url": "http://mail.zope.org/pipermail/zope/2008-August/174025.html"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "http://bugs.gentoo.org/show_bug.cgi?id=246411"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://bugs.launchpad.net/zope2/+bug/257269"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "http://www.zope.org/Products/Zope/Hotfix-2008-08-12/README.txt"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "http://www.zope.org/Products/Zope/Hotfix-2008-08-12/Hotfix_20080812-1.1.0.tar.gz"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://bugs.launchpad.net/zope2/+bug/257276"
                },
                {
                    "name": "ADV-2008-2418",
                    "tags": [
                        "vdb-entry",
                        "x_refsource_VUPEN"
                    ],
                    "url": "http://www.vupen.com/english/advisories/2008/2418"
                },
                {
                    "name": "[oss-security] 20081112 CVE Request - Zope 2 - PythonScripts local DoS",
                    "tags": [
                        "mailing-list",
                        "x_refsource_MLIST"
                    ],
                    "url": "http://openwall.com/lists/oss-security/2008/11/12/2"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "cve@mitre.org",
                    "ID": "CVE-2008-5102",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "n/a",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "n/a"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "n/a"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "PythonScripts in Zope 2 2.11.2 and earlier, as used in Conga and other products, allows remote authenticated users to cause a denial of service (resource consumption or application halt) via certain (1) raise or (2) import statements."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "n/a"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "[Zope] 20080812 Script (Python) insecure ?",
                            "refsource": "MLIST",
                            "url": "http://mail.zope.org/pipermail/zope/2008-August/174025.html"
                        },
                        {
                            "name": "http://bugs.gentoo.org/show_bug.cgi?id=246411",
                            "refsource": "CONFIRM",
                            "url": "http://bugs.gentoo.org/show_bug.cgi?id=246411"
                        },
                        {
                            "name": "https://bugs.launchpad.net/zope2/+bug/257269",
                            "refsource": "CONFIRM",
                            "url": "https://bugs.launchpad.net/zope2/+bug/257269"
                        },
                        {
                            "name": "http://www.zope.org/Products/Zope/Hotfix-2008-08-12/README.txt",
                            "refsource": "CONFIRM",
                            "url": "http://www.zope.org/Products/Zope/Hotfix-2008-08-12/README.txt"
                        },
                        {
                            "name": "http://www.zope.org/Products/Zope/Hotfix-2008-08-12/Hotfix_20080812-1.1.0.tar.gz",
                            "refsource": "CONFIRM",
                            "url": "http://www.zope.org/Products/Zope/Hotfix-2008-08-12/Hotfix_20080812-1.1.0.tar.gz"
                        },
                        {
                            "name": "https://bugs.launchpad.net/zope2/+bug/257276",
                            "refsource": "CONFIRM",
                            "url": "https://bugs.launchpad.net/zope2/+bug/257276"
                        },
                        {
                            "name": "ADV-2008-2418",
                            "refsource": "VUPEN",
                            "url": "http://www.vupen.com/english/advisories/2008/2418"
                        },
                        {
                            "name": "[oss-security] 20081112 CVE Request - Zope 2 - PythonScripts local DoS",
                            "refsource": "MLIST",
                            "url": "http://openwall.com/lists/oss-security/2008/11/12/2"
                        }
                    ]
                }
            }
        }
    },
    "cveMetadata": {
        "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "assignerShortName": "mitre",
        "cveId": "CVE-2008-5102",
        "datePublished": "2008-11-17T18:00:00",
        "dateReserved": "2008-11-17T00:00:00",
        "dateUpdated": "2009-09-01T09:00:00",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}